unattended-upgrade

```diff
@@ -1098,14 +1118,21 @@ def conffile_prompt(destFile, prefix=""):
     # in the previous version in the dpkg status file
     if pkg_conffiles:
         for conf_file in pkg_conffiles.split("\n"):
+            installed_conf_file = find_old_installed_conf_file(
+                conf_file, prefix)
             if conf_file not in dpkg_status_conffiles \
-               and os.path.exists(prefix + conf_file):
+               and os.path.exists(installed_conf_file):
                 logging.debug("found conffile %s in new pkg but on dpkg "
                               "status" % conf_file)
                 pkg_md5sum = get_md5sum_for_file_in_deb(destFile, conf_file)
-                with open(prefix + conf_file, 'rb') as fp:
-                    if pkg_md5sum != apt_pkg.md5sum(fp):
-                        return True
+                if os.path.isdir(installed_conf_file):
+                    logging.debug("found conffile %s being a directory on the "
+                                  "system " % conf_file)
+                    pass
```

Comment:
I don't think this `pass` here is needed.

Reply:
Good point, I felt that pass here helps following the logic, but it is indeed obsolete.